### ISSUE TITLE
Landing Page tests via Optimizely to increase share rate

### ISSUE THREAD

**Issue by HPaulJohnson** [MozillaFoundation/encryption_campaign#195]
Let's plan some simple to execute tests that can be deployed via Optimizely to optimize share rate on the /encrypt landing page.

**Comment by HPaulJohnson** [comment]
Ideas to test in this next cycle: 
- Alt headline to "Share this now"
- Alt body copy to "We need your help to spread the word about how encryption protects our privacy."
- Colored share buttons for email/facebook/twitter to make them more visible
- (Possible) replace opt-in in header with share buttons

**Comment by KevZawacki** [comment]
@HPaulJohnson 

Suggested alt copy [here](https://docs.google.com/document/d/138e_Ug-YxN_n5xLEGvXfYBDHhzWBresbnUmwLLlhzRk/edit).

**Comment by HPaulJohnson** [comment]
@lovegushwa can we also consider adding a "share" CTA in the header on this version of the landing page (the one for email traffic): https://advocacy.mozilla.org/encrypt/2/

**Comment by HPaulJohnson** [comment]
@adamlofting I attempted to start setting up these tests in Optimizely today but I'm not seeing our page load correctly in the editor mode. The message I get is "we've loaded your site in compatibility mode..." and I can't override by changing settings in firefox or chrome. Just seeing a blank page in the preview/editor.

Can you take a look at the draft experiment "Sharing Optimization: Encryption Landing Page" and let me know if you have any ideas?

**Comment by adamlofting** [comment]
Some of our sites security settings are blocking Optimizely from loading the site in the editor at the moment. I've filed #202 with a list of the subdomains we need to allow through, and then it should work again.

**Comment by adamlofting** [comment]
@HPaulJohnson the fix for this has just been merged.

@alanmoo can flag once this is deployed to production server.

**Comment by alanmoo** [comment]
You beat me to it in the time I was looking for this issue. It's live on production, propagating across the CDN (within the hour).

**Comment by HPaulJohnson** [comment]
@adamlofting @alanmoo still not working for me - page is blocked in Optimizely editor. 

Perhaps I'm setting up the experiment wrong? What URL should I be using as a basis for experiments?

**Comment by adamlofting** [comment]
I just tried a new experiment and it worked for me here:
<img width="1270" alt="screenshot 2016-03-16 09 20 14" src="https://cloud.githubusercontent.com/assets/1901816/13807318/881fea60-eb58-11e5-81d1-b47124c331be.png">

I tested with this URL: https://advocacy.mozilla.org/encrypt/hybrid/3
But any URL on the site should be OK.

Perhaps try setting it up as a brand new experiment.

**Comment by HPaulJohnson** [comment]
still working out issues with changes being saved in optimizely; have been speaking with their support trying to resolve. 

also, @alanmoo and i are trying to figure out how to use to split traffic to Share Progress share page vs. Share Progress buttons in a modal (this is a P2).

**Comment by alanmoo** [comment]
Have we used Optimizely on other sites built in React before? I'm not sure what your issues are, but based on the way React works I can think of some potential problems.

**Comment by HPaulJohnson** [comment]
@alanmoo, good question, not sure. the odd thing about the basic issue i'm working through with their support is that they are seeing things differently then we are. i'm trying to do some simple tests with variations on copy (changing "share this" text to "help us spread the word"); my changes don't stick after i save and on Optimizely's side they can see the changes (we've been sharing screen shots). It's odd and not sure it's an issue with react (at least in the case of those changes). cc/ @adamlofting

**Comment by adamlofting** [comment]
Hmmm, yes it could be something to do with React. 

donate.mozilla.org is written with React and works with Optimizely, but perhaps it compiles to static HTML? I can't remember. 

https://www.webmaker.org landing page was also React, but again might be simpler than the /encrypt site. That did work with Optimizely.

@alanmoo I've added you to Optimizely as you might be able to spot something in the console in the editor mode.

**Comment by alanmoo** [comment]
I just talked to @alicoding and it doesn't seem likely that this is an issue with React, since this is built very similarly to donate.

If it does turn out to be something on our side, Ali will be the point person on this while I'm out early next week.

**Comment by HPaulJohnson** [comment]
@alanmoo @adamlofting just got off a call with Optimizely support. I was able to resolve the issue in Chrome by installing an extension called "ignore xframe headers" per their suggestion. I tried to install a similar add-on in Firefox but the issue doesn't resolve. 

Ultimately, the changes are being made and this is simply a function of the display in the editor console within Optimizely. If you install the extension above and use Chrome, we should be good to go. 

I have one simple experiment running now to make sure everything is working as expected.
